agent/agent.go

```diff
@@ -503,7 +503,9 @@ func (a *Agent) finishTask(ctx context.Context, tc *taskContext, status string)
 		err := a.uploadToS3(ctx, tc)
 		tc.logger.Execution().Error(errors.Wrap(err, "error uploading log files"))
 		tc.logger.Execution().Infof("Sending final status as: %v", detail.Status)
-		grip.Error(tc.logger.Flush(ctx))
+		flush_ctx, cancel := context.WithTimeout(ctx, 1*time.Minute)
```

Comment:
`time.Minute`, no need to multiply by 1 😄

Reply:
I remove